com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/StepsUtil.java

```diff
@@ -985,23 +985,11 @@ static CloudModelConfiguration getCloudBuilderConfiguration(DelegateExecution co
     }
 
     static String getGitRepoRef(DelegateExecution context) {
-        Object gitRepoConfigObject = context.getVariable(Constants.VAR_GIT_REPOSITORY_CONFIG_MAP);
-        if (gitRepoConfigObject == null) {
-            return (String) context.getVariable(Constants.PARAM_GIT_REF);
-        }
-        @SuppressWarnings("unchecked")
-        Map<String, String> gitRepoConfigMap = (Map<String, String>) gitRepoConfigObject;
-        return gitRepoConfigMap.get(Constants.PARAM_GIT_REF);
+        return (String) context.getVariable(Constants.PARAM_GIT_REF);
     }
 
     static String getGitRepoUri(DelegateExecution context) {
-        Object gitRepoConfigObject = context.getVariable(Constants.VAR_GIT_REPOSITORY_CONFIG_MAP);

```

Comment:
Same.

Reply:
Looool something happened - in my codebase there is no such thing o.O